### PR TITLE
New package: bftpd-4.4

### PR DESCRIPTION
**Pull request by norvegh**

### CODE REVIEW

File: srcpkgs/bftpd/template [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	wget http://sourceforge.net/projects/bftpd/files/bftpd/bftpd-${version}/bftpd-${version}.tar.gz/download
+	if ! sha256sum -c $FILESDIR/sum; then
+		echo "INVALID CHECKSUM"
+		rm -f download
+		return -1
+	fi
+	tar xvfz download
+	rm download
+	mv bftpd bftpd-${version}
+}
```
Comment (xtraeme) [inline srcpkgs/bftpd/template]: Check the `Manual` and specifically for `distfiles` and `checksum`.

You are using wget and do_fetch that is only required in 0.01% real cases.

**Comment by norvegh** [comment]
The purpose is that it is a simple and secure FTP server that actually works out of the box.

File: srcpkgs/bftpd/template [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+maintainer="Norbert Vegh <vegh@norvegh.com>"
+license="GPL-2"
+homepage="http://bftpd.sourceforge.net/"
+checksum=b805ebbdd3de993ca14d733ad9ad007ac342a1646b257d91a53ec87a36c7f741
+distfiles=http://${SOURCEFORGE_SITE}/bftpd/bftpd-${version}.tar.gz
+
+post_extract()
+{
+	cd ${XBPS_BUILDDIR}
+	mv bftpd bftpd-${version}
```
Comment (xtraeme) [inline srcpkgs/bftpd/template]: This `post_extract` is still unnnecessary.

File: srcpkgs/bftpd/files/sum [hunk 3]
```diff
@@ -0,0 +1 @@
+b805ebbdd3de993ca14d733ad9ad007ac342a1646b257d91a53ec87a36c7f741  download
```
Comment (xtraeme) [inline srcpkgs/bftpd/files/sum]: This file is unnecessary...

File: srcpkgs/bftpd/template [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+build_style=configure
```
Comment (xtraeme) [inline srcpkgs/bftpd/template]: And probably you should set `build_style=gnu-configure` and remove `configure_args`.

File: srcpkgs/bftpd/template (hunk 2 above)
Comment (norvegh) [inline srcpkgs/bftpd/template]: The tar.gz extracts into bftpd, without version numbering, so the build process will not find it. How else can I solve this?

Comment (xtraeme) [inline srcpkgs/bftpd/template]: `wrksrc=bftpd`

**Pull request merged**